### PR TITLE
Fix xss in info messages (possible alternative to uicommons pull #29)

### PR DESCRIPTION
**Pull request by isears**

### CODE REVIEW

**Comment by isears** [comment]
This PR is intended to have the same effect as https://github.com/openmrs/openmrs-module-uicommons/pull/29, without unnecessarily escaping all infomessages.

File: omod/src/main/java/org/openmrs/module/registrationapp/page/controller/EditSectionPageController.java [hunk 1]
```diff
@@ -122,8 +122,16 @@ public String post(UiSessionContext sessionContext, PageModel model,
             try {
                 //The person address changes get saved along as with the call to save patient
                 patientService.savePatient(patient);
+
+                // Ensure name is HTML-safe to prevent XSS in info messages
+                PersonName HTMLsafePersonName = new PersonName(
```
Comment (dkayiwa) [inline omod/src/main/java/org/openmrs/module/registrationapp/page/controller/EditSectionPageController.java]: Our convention is to start with lower case for variable names. :)

**Comment by dkayiwa** [comment]
@isears with this alternative, i guess we could revert the other commit in uicommons. Not so? :)

**Comment by isears** [comment]
@dkayiwa Absolutely -- if this PR is accepted the my uicommons contribution that was causing problems should be reverted.

**Comment by dkayiwa** [comment]
@isears do you mind squashing the commits into one? :)

**Comment by isears** [comment]
@dkayiwa done.

**Comment by dkayiwa** [comment]
Thanks @isears 👍

**Pull request merged**